### DISCUSSION TITLE
Welcome to markets Discussions!

### DISCUSSION THREAD

**Discussion by tomasz-oponowicz** [bitstower/markets#44] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edenalencar** [comment]
Hi!
I am a Java developer and I am currently learning Vala and I want to help legal projects like this.
I have a doubt.
Is there any forecast to launch the version with the new Brazilian Portuguese translation?

**Reply by tomasz-oponowicz** [reply]
hi @edenalencar, nice to meet you

The Brazilian translation will be released with [0.6.0 version](https://github.com/bitstower/markets/projects/6).

I expect this to happen in 1-2 weeks.

**Reply by edenalencar** [reply]
Hi @ Tomasz-apanowicz Nice to meet you too!
I'm glad to hear this.
I use markets every day and have been very helpful to me.

**Reply by tomasz-oponowicz** [reply]
@edenalencar I've just released a new version, 0.5.2. It includes Brazilian Portuguese translations. Thanks for your contribution.

**Reply by edenalencar** [reply]
Hi @tomasz-oponowicz!! I appreciate the responsiveness and support for new languages. I'm learning Gtk / Vala and, in the future, I can help markets with code!

**Comment by PedePanoOriginal** [comment]
Hello guys, I really like this app. I would like to give a suggestion but, i don't know if this is just a Brazilian thing. Here we have a called "ordinary stock" that contains a "pack" of 100 stocks from a company but, this pack can be negotiated separately in 100 "fractional stocks". The problem is that they use the same symbol+F at the end to identify this fractional stock, and the name of the company is the same, so we end up with stuff like this:

![image](https://user-images.githubusercontent.com/35506956/136145822-76c813a7-6a15-423a-9798-9662bcd150d0.png)

So if there was an option to switch between companies names and their symbols, we would be able to track better fractional and ordinary stocks. Ticker is an exemple:

![image](https://user-images.githubusercontent.com/35506956/136146877-2bca5289-0488-4f79-aab1-dead9f403941.png)
